src/monster.cpp

```diff
@@ -626,6 +626,23 @@ BlockType_t Monster::blockHit(Creature* attacker, CombatType_t combatType, int32
 				blockType = BLOCK_ARMOR;
 			}
 		}
+		
+		if (const Player* player = attacker->getPlayer()) {
```

Comment:
inverse that condition and early return blockType

Reply:
Wut?

Reply:
Instead of writing bloated if statement you can check for falseness and `return blocktype` inside

Reply:
so it would be something like
```
if (!(const Player* player = attacker->getPlayer()) {
  return blockType;
}
```